### PR TITLE
Add document for manange role by LDAP group(master)

### PR DESCRIPTION
**Pull request by stonezdj**
Document features related to EPIC #3506 Support to assign role to LDAP group in project level

### CODE REVIEW

**Comment by reasonerjt** [comment]
It would be better if we can reference this doc from user doc md file.

**Comment by hainingzhang** [comment]
does it make more sense to put this doc as a section in the user guide?

File: docs/manage_role_by_ldap_group.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   * ldap_group_gid    -- The attribute used to name an LDAP/AD group, for example: cn 
+   * ldap_group_scope  -- The scope to search for LDAP/AD groups. 0-LDAP_SCOPE_BASE, 1-LDAP_SCOPE_ONELEVEL, 2-LDAP_SCOPE_SUBTREE  
+   * ldap_group_admin_dn -- Specify the LDAP group DN with harbor admin privilege.
+
+  2. Or Change configure parameter in web console after installation. Go to "Administration" -> "Configuration" -> "Authentication" and change following settings.
+   - LDAP Group Base DN -- ldap_group_basedn in harbor.cfg
+   - LDAP Group Filter  -- ldap_group_filter in harbor.cfg
+   - LDAP Group GID     -- ldap_group_gid in harbor.cfg
+   - LDAP Group Scope   -- ldap_group_scope in harbor.cfg
+   - LDAP Groups With Admin Privilege -- ldap_group_admin_dn in harbor.cfg
```
Comment (reasonerjt) [inline docs/manage_role_by_ldap_group.md]: Maybe we need more explanation for this attribute, as it is for a slightly different use case.

File: docs/manage_role_by_ldap_group.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Besides **[basic LDAP configure paremters](https://github.com/vmware/harbor/blob/master/docs/installation_guide.md#optional-parameters)** , LDAP group related configure parameters should be configured, they can be configured before or after installation
+
+  1. Configure parameters in harbor.cfg before installation
+
+   * ldap_group_basedn -- The base DN from which to lookup a group in LDAP/AD, for example: ou=groups,dc=example,dc=com
+   * ldap_group_filter -- The filter to search LDAP/AD group, for example: objectclass=groupOfNames 
+   * ldap_group_gid    -- The attribute used to name an LDAP/AD group, for example: cn 
+   * ldap_group_scope  -- The scope to search for LDAP/AD groups. 0-LDAP_SCOPE_BASE, 1-LDAP_SCOPE_ONELEVEL, 2-LDAP_SCOPE_SUBTREE  
+   * ldap_group_admin_dn -- Specify the LDAP group DN with harbor admin privilege.
```
Comment (reasonerjt) [inline docs/manage_role_by_ldap_group.md]: I think this is not in harbor.cfg?

**Comment by clouderati** [comment]
lgtm

**Comment by reasonerjt** [comment]
@stonezdj please rebase.

**Pull request merged**